packages/Core/theme/checkboxes.ts

```diff
@@ -9,15 +9,15 @@ export type ThemeCheckboxes = {
 }
 
 export const getCheckboxes = (theme: WuiTheme): ThemeCheckboxes => {
-  const { colors } = theme
+  const { colors, toRem } = theme
   return {
     default: {
-      width: 20,
-      height: 20,
+      width: toRem(16),
+      height: toRem(16),
```

Comment:
Use size from theme ? (space.lg)

Reply:
Not here, is a size of the component no a spacing